CI: tests/scripts/validate_cluster.sh: line 60: display_status: command not found

canary test `encryption-pvc-db` failing with error
```
tests/scripts/validate_cluster.sh: line 60: display_status: command not found
+ return current status: cluster: id: ced4347f-aa20-4936-8bfb-50d6f8e139d6 health: HEALTH_WARN Reduced data availability: 1 pg inactive OSD count 1 '<' osd_pool_default_size 3 services: mon: 1 daemons, quorum a '(age' '4m)' mgr: 'a(active,' since '4m)' osd: 1 osds: 0 up, 0 in data: pools: 1 pools, 1 pgs objects: 0 objects, 0 B usage: 0 B used, 0 B / 0 B avail pgs: 100.000% pgs unknown 1 unknown
tests/scripts/validate_cluster.sh: line 60: return: current: numeric argument required
++ display_status
tests/scripts/validate_cluster.sh: line 130: display_status: command not found
Error: Process completed with exit code 127.
```

example run https://github.com/rook/rook/pull/8613/checks?check_run_id=3940786175

--- a/tests/scripts/validate_cluster.sh
+++ b/tests/scripts/validate_cluster.sh
@@ -27,8 +27,6 @@ OSD_COUNT=$2
 #############
 EXEC_COMMAND="kubectl -n rook-ceph exec $(kubectl get pod -l app=rook-ceph-tools -n rook-ceph -o jsonpath='{.items[*].metadata.name}') -- ceph --connect-timeout 3"
 
-trap display_status SIGINT ERR
-
 function wait_for_daemon () {
   timeout=90
   daemon_to_test=$1

